extensions/openapi-generator/deployment/src/test/resources/templates/hello.qute

```diff
@@ -0,0 +1 @@
+Hello! My name is {name}
```

Comment:
is this used?

Reply:
Just for the initial "test first/validate the templating engine". I'll change it in the final version.